cuda/matrix/fbcsr_kernels.cu

```diff
@@ -57,11 +83,55 @@ namespace cuda {
 namespace fbcsr {
 
 
+constexpr int default_block_size{512};
+
+
+#include "common/components/uninitialized_array.hpp.inc"
+#include "common/matrix/fbcsr_kernels.hpp.inc"
+
+
 template <typename ValueType, typename IndexType>
 void spmv(std::shared_ptr<const CudaExecutor> exec,
           const matrix::Fbcsr<ValueType, IndexType> *a,
-          const matrix::Dense<ValueType> *b,
-          matrix::Dense<ValueType> *c) GKO_NOT_IMPLEMENTED;
+          const matrix::Dense<ValueType> *b, matrix::Dense<ValueType> *c)
+{
+    if (cusparse::is_supported<ValueType, IndexType>::value) {
+        auto handle = exec->get_cusparse_handle();
+        {
```

Comment:
I'm not completely sure. This is how it's done for the Csr cusparse calls too. Maybe it's to ensure the `pointer_mode_guard` is destroyed before the handle? Otherwise, I don't think there's a strict ordering between destruction of `handle` and `pm_guard`.

Reply:
Okay, turns out it's unnecessary; I removed it.